website/www/site/content/en/documentation/_index.md

```diff
@@ -22,42 +23,38 @@ limitations under the License.
 
 This section provides in-depth conceptual information and reference material for the Beam Model, SDKs, and Runners:
 
+</br>
+
 ## Concepts
 
 Learn about the Beam Programming Model and the concepts common to all Beam SDKs and Runners.
 
-* Read the [Programming Guide](/documentation/programming-guide/), which introduces all the key Beam concepts.
-* Learn about Beam's [execution model](/documentation/runtime/model) to better understand how pipelines execute.
-* Visit [Learning Resources](/documentation/resources/learning-resources) for some of our favorite articles and talks about Beam.
+- Read the [Programming Guide](/documentation/programming-guide/), which introduces all the key Beam concepts.
+- Learn about Beam's [execution model](/documentation/runtime/model) to better understand how pipelines execute.
+- Visit [Learning Resources](/documentation/resources/learning-resources) for some of our favorite articles and talks about Beam.
 
 ## Pipeline Fundamentals
 
-* [Design Your Pipeline](/documentation/pipelines/design-your-pipeline/) by planning your pipeline’s structure, choosing transforms to apply to your data, and determining your input and output methods.
-* [Create Your Pipeline](/documentation/pipelines/create-your-pipeline/) using the classes in the Beam SDKs.
-* [Test Your Pipeline](/documentation/pipelines/test-your-pipeline/) to minimize debugging a pipeline’s remote execution.
+- [Design Your Pipeline](/documentation/pipelines/design-your-pipeline/) by planning your pipeline’s structure, choosing transforms to apply to your data, and determining your input and output methods.
+- [Create Your Pipeline](/documentation/pipelines/create-your-pipeline/) using the classes in the Beam SDKs.
+- [Test Your Pipeline](/documentation/pipelines/test-your-pipeline/) to minimize debugging a pipeline’s remote execution.
```

Comment:
It looks like you've just changed the bullets from * to - here. Could we just leave it as-is to avoid unnecessary content changes?

Reply:
I will update all content files in all pr's manually at monday and tuesday so this things will be resolved.

Reply:
It looks like this * to - change still needs to be reverted.

Reply:
reverted

Reply:
Thank you!